DevoxxClientMobile/src/main/java/com/devoxx/views/ConfSelectorPresenter.java

```diff
@@ -119,20 +127,30 @@ public void initialize() {
 
     private MenuPopupView getMenuPopupView(Button filter) {
         Menu menu = new Menu();
-        MenuItem devoxx = new MenuItem(Conference.Type.DEVOXX.name());
-        MenuItem voxxed = new MenuItem(Conference.Type.VOXXED.name());
-        menu.getItems().addAll(devoxx, voxxed);
+        MenuItem devoxx = new MenuItem(Conference.Type.DEVOXX.toString());
```

Comment:
It would be nice to have an indication of the current selection. As of now the three options look the same, and all can be selected.

Reply:
We already have the label on the top of the view, showing current selection. Adding a selection style to the MenuItem would actually look odd. I don't remember seeing current selection highlighting in apps which follow MD specs.

Reply:
ok, it is not a big deal.